docs/build/nodes/evm-tracing-node.md

```diff
@@ -18,13 +18,7 @@ Requirements for running a tracing node are similar to what we recommend for an
 
 ## Node launch 
 
```

Comment:
One suggestion from me - acknowledge that before a separate binary had to be used, but that hasn't been the case since `v5.44.0` release. Just in case people get confused.

Minor suggestions though, you can ignore it if you wish.

Reply:
Yeah makes sense

Reply:
@Dinonard noted